2/README.md

```diff
@@ -54,29 +54,62 @@ There are X main use cases for Template:
 
 A Tango Device is a strict definition of a distributed object. 
 * configuration are represented in the form of properties.
-* data are represented in the form of attributes.
+* data are represented in the form of attributes and pipes.
 * actions are represented in the form of commands.
 
 Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
 
 
 * The Device is a distributed object which SHALL be accessed locally or via network.
-* The Device SHALL be an instance of one Class
+* The Device SHALL be an instance of one Device Class (see RFC-9)
 * The Device MAY have one or several Property, called Device Property
-* The Device MAY have one or several Attribute
+* The Device MAY have one or several Attribute, 
 * The Device SHALL have one State attribute 
```

Comment:
This seems to be missing:
The device SHALL have one State command
The device SHALL have one Status command

Reply:
As far as I remember, State and Status Commands are sort of legacy stuff, before they got properly implemented as Attributes. Indeed, State and Status Commands are in the current model, thus showing up in Tango 9 RFC, but I'm not sure they need to be there for Tango 10. Your thoughts? Does it make sense to capture in a note?

Reply:
> Indeed, State and Status Commands are in the current model, thus showing up in Tango 9 RFC, but I'm not sure they need to be there for Tango 10.

My comment was indeed only for Tango 9 RFC. I think we should get rid of State and Status mandatory commands in "Tango 10".

Reply:
Thanks, @bourtemb and @lorenzopivetta. The below lines add the missing commands and note on future plans.

Reply:
@lorenzopivetta @bourtemb 
Can you make an issue for the commands to remove with the tag v10?